### PR TITLE
[DCA] fix time based DCA

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

**Review by Herklos — COMMENTED** [review]
:open_mouth: Should we add a simple test on it ?

**Comment by GuillaumeDSM** [comment]
> 😮 Should we add a simple test on it ?

Yes, good idea, i will !

**Comment by GuillaumeDSM** [comment]
up

File: Trading/Mode/dca_trading_mode/tests/test_dca_trading_mode.py [hunk 1]
```diff
@@ -210,6 +211,65 @@ async def test_init_config_values(tools):
...
+
+        # time based
+        mode.trigger_mode = dca_trading.TriggerMode.TIME_BASED
+        await producer.inner_start()
+        for _ in range(10):
+            await asyncio_tools.wait_asyncio_next_cycle()
+        dca_task_mock.assert_called_once()
+
+
+async def test_dca_task(tools):
```
Comment (Herklos) [inline Trading/Mode/dca_trading_mode/tests/test_dca_trading_mode.py:233]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**